source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -55,57 +55,160 @@
 	},
 	"containerStyles": {
 		"default": {
-			"fontColors": {
+			"foregroundColors": {
 				"default": {
-					"normal": "#333333",
+					"default": "#333333",
 					"subtle": "#EE333333"
 				},
 				"accent": {
-					"normal": "#2E89FC",
+					"default": "#2E89FC",
 					"subtle": "#882E89FC"
 				},
 				"good": {
-					"normal": "#54a254",
+					"default": "#54a254",
 					"subtle": "#DD54a254"
 				},
 				"warning": {
-					"normal": "#c3ab23",
+					"default": "#c3ab23",
 					"subtle": "#DDc3ab23"
 				},
 				"attention": {
-					"normal": "#FF0000",
+					"default": "#FF0000",
 					"subtle": "#DDFF0000"
 				}
 			},
 			"backgroundColor": "#EAEAEA"
 		},
 		"emphasis": {
-			"fontColors": {
+			"foregroundColors": {
 				"default": {
-					"normal": "#333333",
+					"default": "#333333",
 					"subtle": "#EE333333"
 				},
 				"accent": {
-					"normal": "#2E89FC",
+					"default": "#2E89FC",
 					"subtle": "#882E89FC"
 				},
 				"good": {
-					"normal": "#54a254",
+					"default": "#54a254",
 					"subtle": "#DD54a254"
 				},
 				"warning": {
-					"normal": "#c3ab23",
+					"default": "#c3ab23",
 					"subtle": "#DDc3ab23"
 				},
 				"attention": {
-					"normal": "#FF0000",
+					"default": "#FF0000",
 					"subtle": "#DDFF0000"
 				}
 			},
 			"backgroundColor": "#08000000",
 			"borderColor": "#FF000000",
 			"borderThickness": 2
-			}
+		},
+		"good": {
+			"foregroundColors": {
+				"default": {
+					"default": "#333333",
+					"subtle": "#EE333333"
+				},
+				"accent": {
+					"default": "#2E89FC",
+					"subtle": "#882E89FC"
+				},
+				"good": {
+					"default": "#54a254",
+					"subtle": "#DD54a254"
+				},
+				"warning": {
+					"default": "#c3ab23",
+					"subtle": "#DDc3ab23"
+				},
+				"attention": {
+					"default": "#FF0000",
+					"subtle": "#DDFF0000"
+				}
+			},
+			"backgroundColor": "#DD54a254"
+		},
+		"warning": {
+			"foregroundColors": {
+				"default": {
+					"default": "#333333",
+					"subtle": "#EE333333"
+				},
+				"accent": {
+					"default": "#2E89FC",
+					"subtle": "#882E89FC"
+				},
+				"good": {
+					"default": "#54a254",
+					"subtle": "#DD54a254"
+				},
+				"warning": {
+					"default": "#c3ab23",
+					"subtle": "#DDc3ab23"
+				},
+				"attention": {
+					"default": "#FF0000",
+					"subtle": "#DDFF0000"
+				}
+			},
+			"backgroundColor": "#DDc3ab23"
+		},
+		"attention": {
+			"foregroundColors": {
+				"default": {
+					"default": "#333333",
+					"subtle": "#EE333333"
+				},
+				"accent": {
+					"default": "#2E89FC",
+					"subtle": "#882E89FC"
+				},
+				"good": {
+					"default": "#54a254",
+					"subtle": "#DD54a254"
+				},
+				"warning": {
+					"default": "#c3ab23",
+					"subtle": "#DDc3ab23"
```

Comment:
out of curiosity, why are we mixing case here?

Reply:
Didn't realize it's not case sensitive. Oops!

---
In reply to: [257394613](https://github.com/Microsoft/AdaptiveCards/pull/2289#discussion_r257394613) [](ancestors = 257394613)